### DISCUSSION TITLE
How can I implement a form validation without using the form label?

### DISCUSSION THREAD

**Discussion by hellohejinyu** [react-hook-form/react-hook-form#12344] [Q&A]
I found that Chrome has problems rendering forms in certain layouts. So I decided to replace the `form` with a `div`.

![WX20241018-141449@2x](https://github.com/user-attachments/assets/c7f42a7f-011d-4af8-ae09-88214ccb12c0)

I didn't find any relevant examples in the documentation.

I have achieved my goal through `trigger` and `getValues` ​​methods, but I am not sure if there are any potential problems. I am unfamiliar with `react-hok-form`, so I hope a professional developer can answer this question.

**Comment by Arshad1071** [comment]
Input type of this name is focusing on it, how can I name it like this without focusing and changing the name?  

 <input
                id={`user_members.${index}.relationship`}
                name={`user_members.${index}.relationship`}
                className="block p-1 text-sm text-black resize-none   md:w-40 w-full h-[30px] rounded-[5px] border border-[#7d7d87]"
                placeholder=""
                {...register(`user_members.${index}.relationship`)}
              />
